isthmus/src/test/java/io/substrait/isthmus/SubstraitRelNodeConverterTest.java

```diff
@@ -285,4 +285,24 @@ public void emit() {
       assertRowMatch(relNode.getRowType(), R.I32, N.STRING);
     }
   }
+
+  @Nested
+  class Switch {
+    @Test
+    public void simple() {
+      Plan.Root root =
+          b.root(
+              b.project(
+                  input ->
+                      List.of(
+                          b.switchExpression(
+                              b.fieldReference(input, 0),
+                              List.of(b.switchClause(b.i32(0), b.fieldReference(input, 3))),
+                              b.bool(false))),
+                  b.remap(4),
+                  commonTable));
+      var relNode = converter.convert(root.getInput());
+      assertRowMatch(relNode.getRowType(), N.BOOLEAN);
+    }
+  }
```

Comment:
This test feel out of place in this file, given that it's testing an expression conversion rather an a rel conversion. It also seem redundant as your other test is more thorough.